scripts/check_frontend_coverage.py

```diff
@@ -505,14 +503,13 @@
     'thumbnail-uploader.directive.ts',
     'top-navigation-bar.directive.ts',
     'topic-creation.service.ts',
+    'topic-editor-helper.service.ts',
```

Comment:
Why is this being added here?

Reply:
Will remove this when skills dashboard PR goes in.

Reply:
OK, please loop her in too when you do things like this and get explicit confirmation from her, now and in the future. /cc @marianazangrossi

Reply:
I think it's okay to add in the list if in the future you plan to test the service you're adding now.